docs/plugins/survey.md

```diff
@@ -42,169 +28,203 @@ Other parameters can be left unspecified if the default value is acceptable.
+For reference, the SurveyJS question/element types are listed below, with links to more information and examples in the SurveyJS documentation. 
 
-Parameter | Type | Default Value | Description
-----------|------|---------------|------------
-type | string | *undefined* | The question type. Options are: "drop-down", "html", "likert", "likert-table", "multi-choice", "multi-select", "ranking", "rating", "text".
-prompt | string | *undefined* | The prompt/question that will be associated with the question's response field. If the question type is "html", then this string is the HTML-formatted string to be displayed. If the question type is "likert-table", the prompt is a general question or title presented above the table.
-required | boolean | `false` | Whether a response to the question is required (`true`) or not (`false`), using the HTML5 `required` attribute. 
-name | string | `null` | Name of the question to be used for storing data. If this parameter is not provided, then default names will be used to identify the questions in the data: `P0_Q0`, `P0_Q1`, `P1_Q0` etc. Question names must be unique across all pages within the trial.
+#### boolean
 
-#### Drop-down
+The "boolean" type is a yes/no (or other two-option) multiple choice question. It differs from the "radiogroup" (multiple choice) question type in that it offers different response format options (left/right slider button, yes/no checkbox) in addition to the standard multiple choice format (two separate radio buttons).
 
-Present a question with a limited set of options in a drop-down menu. The participant can only select one option.
+- [Boolean example](https://surveyjs.io/form-library/examples/yes-no-question/jquery) 
+- [Boolean API documentation](https://surveyjs.io/form-library/documentation/api-reference/boolean-question-model)
 
-In addition to the parameters for all question types, the drop-down question type also offers the following parameters. 
-Parameters with a default value of *undefined* must be specified.
-Other parameters can be left unspecified if the default value is acceptable.
+#### checkbox
 
-Parameter | Type | Default Value | Description
-----------|------|---------------|------------
-options | array of strings | *undefined* | This array contains the set of multiple choice options to display for the question.
-option_reorder | string | "none" | One of: "none", "asc", "desc", "random". If "none", the options will be listed in the order given in the `options` array. If `random`, the option order will be randomized. If "asc" or "desc", the options will be presented in ascending or descending order, respectively.
-correct_response | string | null | String from the `options` array that should be considered correct. If specified, the data will include a `correct` property that indicates whether the response was correct (`true`) or not (`false`).
+This is the "multi-select" question type: participants can select one or more options from the set of choices.
```

Comment:
I don't know if we should reference existing jsPsych survey naming here. Future users will probably start with this one. And checkbox is more familiar than multi-select.

Reply:
Makes sense. Fixed!